Classes/Utility.php

```diff
@@ -65,9 +65,10 @@ public function __construct(ConfigurationReader $configuration) {
 	 *
 	 * @param string $string
 	 * @param string $spaceCharacter
+	 * @param bool $strToLower
 	 * @return string
 	 */
-	public function convertToSafeString($string, $spaceCharacter = '-') {
+	public function convertToSafeString($string, $spaceCharacter = '-', $strToLower = true) {
 		$processedTitle = $this->csConvertor->conv_case('utf-8', $string, 'toLower');
```

Comment:
Because of `specCharsToASCII` on line 74.

This code is historical, from the old version. I am not sure why there is this double conversion but I do not want to change it without a real reason.